app/models/post.rb

```diff
@@ -1,2 +1,5 @@
 class Post < ApplicationRecord
```

Comment:
- [ ] OPTIONAL
It is a nice idea to use one of the Rails Active Callbacks `after_save` to call the `update_posts_counter method every time a new like has been saved to database, then make the `update_posts_counter` method private, similar to:

```ruby
 # app/models/posts.rb
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```

Reply:
Done